windows-start.bat

```diff
@@ -0,0 +1,15 @@
+@echo off
+REM Set the USDROOT environment variable to the relative path
+set USDROOT=.\build\usd
+
+REM Set the PYTHON_DIR to the Python directory in the USD build
+set PYTHON_DIR=%USDROOT%\python
+
+REM Update the PATH variable to include USD binaries, libraries, and Python
+set PATH=%PATH%;%USDROOT%\bin;%USDROOT%\lib;%PYTHON_DIR%;%PYTHON_DIR%\Scripts
```

Comment:
I would put the `%PATH%` variable at the end so the `%USDROOT%\lib` and `%USDROOT%\bin` are picked up before what is in the PATH. I had a USD lib in the path and it was conflicting. Doing this change fixed the conflict.